### DISCUSSION TITLE
[Setting][mWeb] Can’t Scroll The Pages when i want to change my repository visibility

### DISCUSSION THREAD

**Discussion by destafajri** [community/community#51072] [Repositories] [answered]
### Select Topic Area

Bug

### Body

https://user-images.githubusercontent.com/106084465/227799130-bfdbe954-1fcf-42f5-87d0-0ea9f9aa2680.MOV

I Can’t Scroll The Pages on the mweb when i want to change my repository visibility..
My Device is iPhone 7 and using chrome for my browser..

Step to reproduce..

Pre-Condition: login as user via mobile web and has active repository

Step:
1. Go to detail repository
2. Select setting
3. Scroll down into danger zone section
4. Tap change visibility

Notes: user can’t scroll the pages to change the repository visibility

**Comment by iskae** [comment, marked as answer]
Hello @destafajri 👋

Thanks a bunch for reaching out, but unfortunately the Mobile category is only related to GitHub Mobile Apps and not the browser on mobile devices 😅 You may have a better chance if you reach out to our colleagues in [Repositories](https://github.com/orgs/community/discussions/categories/repositories?discussions_q=is%3Aopen+sort%3Adate_created+is%3Aunanswered+category%3ARepositories) for this 🙇

Cheers!
